pynicotine/plugins/core_commands/__init__.py

```diff
@@ -19,6 +19,7 @@
 from pynicotine.pluginsystem import BasePlugin
 from pynicotine.shares import PermissionLevel
 from pynicotine.slskmessages import UserStatus
+from pynicotine.utils import humanize
```

Comment:
Not required here because there are unlikely to be over 1000 custom plugin commands. If there are that many, then the user has larger problems to worry about other than the proper localization of numerals in the help!